tutorials/helloworld/xebialabs/values.xlvals

```diff
@@ -0,0 +1 @@
+# This file includes all non-secret values, you can add variables here and then refer them with '!value' tag in YAML files
```

Comment:
this file will be autogenerated no need to add here

Reply:
Ok.